src/guides/v2.3/javascript-dev-guide/javascript/requirejs.md

```diff
@@ -68,6 +68,24 @@ paths: {
 }
 ```
 
+{:.bs-callout-info}
+If you set a path to an array, and define multiple script sources. If the first script fails to load, the next one will be used as a fallback.
+
+```javascript
+var config = {
+    ...
+    paths: {
+        'alias': [
+            'https://some-library.com/file',
+            'vendor_name>_<module_name>/js/file'
+        ]
+    }
+};
+```
+
+For external content resources should be whitelist, Otherwise Magento will raise error notices in browser console. Reference [here]({{ page.baseurl }}/extension-dev-guide/security/content-security-policies.html)
```

Comment:
```suggestion
For external content resources should be whitelisted, otherwise Magento will raise error notices in the browser console. Reference [Content Security Policies]({{ page.baseurl }}/extension-dev-guide/security/content-security-policies.html).
```